contracts/mock_contract_u64/src/lib.rs

```diff
@@ -40,6 +43,7 @@ pub fn query(_deps: Deps, _env: Env, msg: QueryMsg) -> StdResult<Binary> {
     match msg {
         QueryMsg::FirstQuery {} => to_binary("first query passed"),
         QueryMsg::SecondQuery { .. } => Err(StdError::generic_err("Query not available")),
+        QueryMsg::ThirdQuery {} => to_binary("third query passed"),
```

Comment:
```suggestion
        QueryMsg::ThirdQuery => to_binary("third query passed"),
```

Reply:
Fixed in https://github.com/AbstractSDK/cw-orchestrator/pull/196/commits/06214ce275c7f72419194d45147c8ac0ec1c86ed